builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1076,6 +1110,67 @@ def LoadConfiguration() {
 	return prop
 }
 
+/** 
+** Create Serverless.yml from config
+**/
+
+def loadServerlessYml(config, env){
+	def deploymentDescriptor = prepareServerlessYml(config, env) // Generating the deployment descriptor
+	echo "prepareServerlessYml = ${deploymentDescriptor}"
+	writeYaml(file: './serverless.yml', data: deploymentDescriptor)
+}
+
+/**
+* Json representations of 
+* serverless.yml
+**/
+def prepareServerlessYml(aConfig, env) {
```

Comment:
Since the same method we are using in build-pack-sls-app and delete workflow, please see whether we can write it to a common module, let say, sbr.groovy or so.

Reply:
Updated and added the prepareServerlessYml function on sbr.groovy.